libcloud/common/digitalocean.py

```diff
@@ -109,6 +109,16 @@ def add_default_headers(self, headers):
         headers['Content-Type'] = 'application/json'
         return headers
 
+    def add_default_params(self, params):
+        """
+        Add parameters that are necessary for every request
+
+        This method adds ``per_page`` to the request to reduce the total
+        number of paginated requests to the API.
+        """
+        params['per_page'] = '200'
```

Comment:
Yes, that may work with the driver constructor. I will review this later today to see how to implement this, as the add_default_params is in the connection class.

I don't know if the connection class ever gets a reference from the driver, though that may actually be passed in as `self`. If that's the case, I could just use a base driver attribute assigned by a keyword parameter in the driver that defaults to 200. I will have to test that later, though if there are any cleaner suggestions, I'm all ears.